### DISCUSSION TITLE
After downgrading Plex, with scrapers working, Men.Com scenes are biting me on WayBig

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#238] [Q&A] [answered]
THANNKS SO MUCH GUYS!!!!!! =DDDDDDD

JASON, THANK YOU!!!! CODY, MAN YOU ROCK!!!!! =DDDD

After adding a bunch of scenes XD I came across a problem with some posts on WayBig

https://www.waybig.com/blog/2022/11/16/men-johnny-donovan-bottoms-for-troye-dean-cumpanion/
https://www.waybig.com/blog/2022/09/26/men-malik-delgaty-fucks-paul-wagner-warm-welcum/
https://www.waybig.com/blog/2022/11/01/men-olivier-robert-felix-fox-and-tayler-tash-cockrise-yoga/

I'm changing MEN.Com to MEN then to Men, then to Men.com, but nothing match them
Then changin the ´´ to '' but again, nothing

Here are the files named right now:
![Screenshot 2023-04-29 123617](https://user-images.githubusercontent.com/20586368/235320325-f4f410c5-7c77-4112-a1d4-9289a9c52c66.jpg)

Once again, thank you Cody and Jason =) for real

**Comment by JPH71** [comment]
I will look at these later this evening....

Quick notes with matching...
Studios will match without .Com,  .net, .Org etc

(Next Door) will match to Next Door Studios, Next Door Twinks etc

(FalconStudios) will match to Falcon Studios

Hope this helps

For matching with waybig I copy the entry... put brackets round the studio,
replace the colon with a minus sign and add the bracketed year to the end...

And drop the .com if they exist

Jason

**Comment by JPH71** [comment]
You seem to have named your files correctly... it has to do with the
apostrophes...

The latest version should fix this...

And remember any time you have an issue ... let's us know xx

Jason xxx

**Reply by CodyBerenson** [reply, marked as answer]
@dimitrimahiro  @JPH71 

Confirmed, the latest version of the agents fixes these three:

![image](https://user-images.githubusercontent.com/40373801/235354400-ca06f4e3-c4e7-4993-b2e3-f4f96d003b3f.png)

**Comment by JPH71** [comment]
Cheers Cody...
I have just home and now have to go for a braai at my friends in another
hour....

I started working on the GEVI scenes....

Jason x

**Comment by dimmitri9** [comment]
AS EXPECTED, IT'S WORKING!!!!!! =DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD

Now I came across something... When a scene is added from Fagalicious blog (which I only use when WayBig doesn't have it), this happens, not only with this title in particular but I tried with other, and same happened

https://fagalicious.com/ragingstallion-ty-roderick-and-jake-genesis-hot-flip-fuck-in-cock-shot/
![Screenshot 2023-05-01 191550](https://user-images.githubusercontent.com/20586368/235547608-44bf5433-8e33-40d9-94ff-ba3c208a4c03.jpg)
![Screenshot 2023-05-01 191610](https://user-images.githubusercontent.com/20586368/235547615-b350ba3e-c0c6-4ae8-b8d0-195f45c97f72.jpg)
![Screenshot 2023-05-01 191632](https://user-images.githubusercontent.com/20586368/235547623-c5e0e001-3348-448d-8769-dc97d31347d4.jpg)

These two 'X' genres act like they are part of the cast =/
So it follows the 'auto-add collection' feature (which I'm fascinated by the way)

Once again, and I'm not tired of saying it, THANKS THANKS THANKS =))))

**Comment by JPH71** [comment]
Thanks for finding those genres that are appearing as cast...
I will add to the gaytidy.txt file and send it to Cody in the morning...

As you find more like these.. just email the genre and I will update the
file....

Cheers Dimitri!!

**Comment by dimmitri9** [comment]
Hey Jason!! =))) Hope you are ok!

Now it's hapeening to show series as cast with WayBig agent
![Screenshot 2023-05-12 121529](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/6a121205-6244-456c-915b-7b8a49662387)

![Screenshot 2023-05-12 121556](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/ed9910e2-4e1c-4093-b71f-fb75fb201133)

![Screenshot 2023-05-12 121604](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/48dc9d56-0193-4099-a1f0-d3efecdfb874)
